Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "rooster"
-version = "2.3.0"
+version = "2.4.0"
```

Comment:
(this changed when I ran `cargo build` -- kept it into a separate commit, but it seems correct given that it matches `Cargo.toml`, right?)